src/main/java/io/openliberty/tools/intellij/actions/LibertyDevCustomStartAction.java

```diff
@@ -63,23 +65,29 @@ protected void executeLibertyAction(LibertyModule libertyModule) {
             // create new run config
             selectedLibertyConfig = createNewLibertyRunConfig(runManager, libertyModule);
         } else {
-            // TODO if 1+ run configs, prompt user to select the one they want see https://github.com/OpenLiberty/liberty-tools-intellij/issues/167
             // 1+ run configs found for the given project
-            RunnerAndConfigurationSettings selectedConfig = runManager.getSelectedConfiguration();
-            if (libertyModuleSettings.contains(selectedConfig)) {
-                // if the selected config is for the Liberty module, use that run config
-                selectedLibertyConfig = selectedConfig;
+            final String[] runConfigNames = toRunConfigNames(libertyModuleSettings);
+            final String[] runConfigNamesTooltips = toRunConfigNamesTooltips(libertyModuleSettings);
```

Comment:
```suggestion
            final String[] runConfigNamesTooltips = new String[libertyModuleSettingssize())];
```
Can we pass an empty array in LibertyProjectChooserDialog constructor? I think if we pass like that there won't be any tooltips. As per current code we are showing not showing any additional code in tooltip.